third_party/CMakeLists.txt

```diff
@@ -13,6 +13,26 @@ if (BUILD_SOX)
   list(APPEND TORCHAUDIO_THIRD_PARTIES libsox)
 endif()
 
+################################################################################
+# ffmpeg
+################################################################################
+if (BUILD_FFMPEG)
+  find_package(PkgConfig REQUIRED)
+  pkg_check_modules(LIBAV REQUIRED IMPORTED_TARGET
+    # requires ffmpeg>=4.1
```

Comment:
Is the comment intended?

Reply:
Yes. ffmpeg *package* is composed of multiple libraries, each of which has diff version numbers. https://ffmpeg.org/download.html#release_4.1

Without a comment It is hard to see which version of ffmpeg package is required here.